test/feature/support/helpers.coffee

```diff
@@ -19,12 +19,21 @@ class Helpers
       app.use exprestive(#{optionsStr})
       app.listen #{@port}
 
+      # Route to evaluate functions from tests
+      app.get '/eval/:strToEval', (req, res) ->
```

Comment:
It would be nice if exprestive allowed to specify flexible routes like that as well.